kymatio/scattering1d/frontend/__init__.py

```diff
@@ -0,0 +1,19 @@
+__all__ = ['Scattering1D']
+
+import logging
+
+class Scattering1D(object):
```

Comment:
This file is very similar to the one in scattering2d.

Reply:
maybe should it be shared?

Reply:
How do we wanna do this?

Do we wanna define a subclass `ScatteringBase` in the main `kymatio` folder, from which `Scattering*D` subclass inherits from, and all we do here is init the `ScatteringBase`? 

@lostanlen @janden your thoughts are also appreciated.

Reply:
That would make sens: code redundancy can elegantly be avoided thanks to oop